members/SWIA357_369417.yaml

```diff
@@ -0,0 +1,53 @@
+bioguide: SWIA357_369417
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "https://www.doj.state.wi.us/ag/contact"
+    - fill_in:
+        - name: First Name
+          selector: input#edit-submitted-contact-first-name
+          value: $NAME_FIRST
+          required: true
+        - name: Last Name
+          selector: input#edit-submitted-contact-last-name
+          value: $NAME_ LAST
+          required: true
+        - name: Street
+          selector: input#edit-submitted-contact-street-address
+          value: $ADDRESS_STREET
+          required: true
+        - name: City
+          selector: input#edit-submitted-contact-city
+          value: $ADDRESS_CITY
+          required: true
+        - name: State
+          selector: input#edit-submitted-contact-state
+          value: Wisconsin
+          required: true
+        - name: Zip
+          selector: input#edit-submitted-contact-zip-code
+          value: $ADDDRESS_ZIP5
+          required: true
+        - name: Phone
+          selector: input#edit-submitted-contact-daytime-phone
+          value: $PHONE
+          required: true
+        - name: email
+          selector: input#edit-submitted-contact-email-address
+          value: $EMAIL
+          required: true
+        - name: Message
+          selector: textarea#edit-submitted-contact-type-your-message
+          value: $MESSAGE
+          required: true
+    - click_on:
+        - value: Submit
+          selector: input[type='submit'][value='Submit']
+    - wait:
+        - value: 3
```

Comment:
Is it possible to use a `find` instead? That will wait a maximum of 5 seconds but will take less time if it finds it sooner